src/atomate2/forcefields/jobs.py

```diff
@@ -322,7 +410,6 @@ class GAPRelaxMaker(ForceFieldRelaxMaker):
     relax_cell: bool = False
     steps: int = 500
     relax_kwargs: dict = field(default_factory=dict)
-    optimizer_kwargs: dict = field(default_factory=dict)
```

Comment:
deleted as unused, may be passed in somewhere in a flow which may necessitate reverting this.

Reply:
It is used in Line 82, I think.

Reply:
will it not inherit from the base class in that instance however?

Reply:
Yes, sure, but we typically mention those as well in atomate2.

Reply:
will put back